.github/workflows/build_extension_deb.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Extension .deb
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build-deb:
+    # source: https://github.com/supabase/pg_jsonschema/blob/29fcd5b23abe3dd6a78a490082f34113de7813a1/.github/workflows/release.yml
+    name: release .deb
+    if: github.event_name == 'release'
+    strategy:
+      matrix:
+        extension_name:
+          - pgmq
+        package_name:
+          - pgmq
+        pgrx_version:
+          - 0.11.3
```

Comment:
Consider reading it from the `Cargo.toml` file so you don't have to remember to change it here.